examples/minimal-content-types/client/src/SendMultipartFormDataButton.tsx

```diff
@@ -6,12 +6,19 @@ export function SendMultipartFormDataButton() {
   return (
     <button
       onClick={() => {
-        const data = new FormData();
+        const fd = new FormData();
 
-        data.set('name', 'John Doe');
-        data.set('occupation', 'tRPC Extraordinaire');
+        fd.set('name', 'John Doe');
+        fd.set('occupation', 'tRPC Extraordinaire');
 
-        mutation.mutate(data);
+        fd.set(
+          'about',
+          new File(['hi bob'], 'bob.txt', {
+            type: 'text/plain',
+          }),
+        );
```

Comment:
cc @Nick-Lucas 

I don't see why you'd want to use octet streams over `FormData`? 

If you upload a file you can just do this?

Reply:
saying this because it seems a lot easier to deal with a `File` than a `ReadableStream`

Reply:
Choice is good, not everything is a File and you might also want to stream a file direct to some storage which I don't believe FormData could do, it would have to fit in memory